doc/theoretical_description_regression.rst

```diff
@@ -245,30 +245,38 @@ uncertainty is higher than :math:`CV+`, because the models' prediction spread
 is then higher.
 
 
-9. The conformalized quantile regression (CQR) method
-=====================================================
+9. The Conformalized Quantile Regression (CQR) Method
+==================================================
```

Comment:
Add as many "=" characters as there are characters in the previous line.

Reply:
done